draft-ietf-tls-tls13.md

```diff
@@ -2612,6 +2612,12 @@ facilitates implementation as described below). Servers MUST check
 that it is the last extension and otherwise fail the handshake with an
 "illegal_parameter" alert.
 
+If a server fails to retrieve a PSK value from identity values
+(because of some configuration or state changes, for instance), the
+server cannot resume the previous session nor verify the binders. In
+this case, the server SHOULD fall back to a full 1-RTT handshake if a
+"key_share" extension is supplied.
```

Comment:
It might be a good idea to note that if the keys CAN be recovered, failing to verify the binders is fatal.